apps/monitoring/src/pages/configuration/[[...path]].tsx

```diff
@@ -50,57 +49,53 @@ const Content = () => {
     const site = configuration.data?.integrations.datadog?.credentials.site ?? DatadogSite.Us1;
 
     return (
-      <Section>
-        <Typography paragraph variant="h3">
-          <Done style={{ verticalAlign: "middle", marginRight: 10 }} />
+      <Box>
+        <Text as={"h1"} variant="heading" marginBottom={4}>
           App configured
-        </Typography>
-        <Typography paragraph>
+        </Text>
+        <Text as={"p"}>
           Visit{" "}
-          <Link
-            href="https://app.datadoghq.com/"
-            onClick={(e) => {
-              e.preventDefault();
-              appBridge?.dispatch(
-                actions.Redirect({
-                  to: DATADOG_SITES_LINKS[site],
-                  newContext: true,
-                })
-              );
-            }}
-          >
+          <TextLink newTab href={DATADOG_SITES_LINKS[site] ?? "https://app.datadoghq.com/"}>
             Datadog
-          </Link>{" "}
+          </TextLink>{" "}
           to access your logs
-        </Typography>
-      </Section>
+        </Text>
+        <Button
+          marginTop={4}
+          onClick={() => {
+            push("/configuration/datadog");
+          }}
+        >
+          Edit configuration
+        </Button>
+      </Box>
     );
   }
 
   if (datadogError) {
     return (
-      <Section>
-        <Typography paragraph variant="h3">
-          <Error style={{ verticalAlign: "middle", marginRight: 10 }} />
+      <Box>
+        <Text variant="heading" as={"h1"}>
           Configuration Error
-        </Typography>
-        <Typography>{datadogError}</Typography>
-      </Section>
+        </Text>
+        <Text color={"textCriticalDefault"}>{datadogError}</Text>
+        <Button
+          marginTop={8}
+          onClick={() => {
+            push("/configuration/datadog");
+          }}
+        >
+          Edit configuration
+        </Button>
+      </Box>
     );
   }
 
   return null;
 };
 
 const ConfigurationPage: NextPage = () => {
-  const selectedProvider = useActiveProvider();
-
-  return (
-    <AppColumnsLayout>
-      <IntegrationsList activeProvider={selectedProvider} />
-      <Content />
-    </AppColumnsLayout>
-  );
+  return <Content />;
```

Comment:
Two possibilities, chose one ;) :
- Export content as default since currently it doesn't do anything
- Move data fetching to ConfigurationPage and display loading screen or Content if data are ready

Reply:
Also - according to the screenshot, theres no way go go back besides the browser back button. 
Let's use breadcrumbs to have a similar UI as other apps.